src/encoding/draw.rs

```diff
@@ -104,10 +104,10 @@ pub struct DrawRadialGradient {
 #[derive(Clone, Copy, Debug, Default, Zeroable, Pod)]
 #[repr(C)]
 pub struct DrawImage {
-    /// Image index.
-    pub index: u32,
-    /// Packed image offset.
-    pub offset: u32,
+    /// Packed atlas coordinates.
```

Comment:
Would [u16; 2] be a better choice here? It feels more semantic, but perhaps just using u32 leaves less slop around stuff like endianness.

Reply:
Yeah, it was simply a desire to keep the shape of the data the same for both CPU and GPU. Also, when we do add descriptor indexing, the xy field (maybe renamed location) can be used as the array index in that mode.